### DISCUSSION TITLE
RSP-500-24 misalignment

### DISCUSSION THREAD

**Discussion by TheoBromine2022** [jon-harper/OmniBox#53] [General]
![20221101_200159](https://user-images.githubusercontent.com/111667850/199364442-52b94e3f-2ecc-41f6-b22e-f5f5cab6911f.jpg)

Third hole on the back half of the box has an m4 screw hole that does not line up with the power supply.

![20221101_200345](https://user-images.githubusercontent.com/111667850/199364650-9a0771ec-9a94-42d8-9679-022f3fba49b3.jpg)

Sharpie'd dot is where the hole in the box points to.

After some observations I think the issue is that you have the holes in the right position from each other, they just need to be flipped.

**Comment by jon-harper** [comment]
🤦‍♂️ I guess I had to find out somehow. Thank you for fit testing.

> After some observations I think the issue is that you have the holes in the right position from each other, they just need to be flipped.

I don't know if I want this to be a CAD model error or my own. But that's kinda moot.

What is your thinking on being able to salvage this? Does the PSU mount well enough with three screws? If not, are you able to bore a hole?

If the rear body is not salvageable we can go from there, but I'd at least like to get you up and running for now if I can.

**Comment by TheoBromine2022** [comment]
I think 3 screws is fine for now. I really don't think it's going anywhere
lol.

**Reply by jon-harper** [reply]
That's good news. Thanks again.

**Reply by TheoBromine2022** [reply]
Well I finished one of the more major sidequests of doing a truck rebuild so this project is going to get some attention now. I did some mild test fitting after getting my wire crimps back from the automotive work and I'm going to hopefully get the PSU and octopus set up this weekend. I can tell from the initial test fit that there is... not a whole lot of room available in the bottom for the 500. I think I will edit the monolithic base to have the fan port blow directly into the psu's fan port with a noctua that is always on with the hopes that it keeps the psu fan from kicking on. really easy edit but it might throw off the feng shui of the front and I doubt I can get a 40mm filter to fit in there well. I need to find a small mesh screen and try that.

**Reply by jon-harper** [reply]
Re: the RSP-500

That tight fit is what actually gave me the idea for the front vents. They weren't there in 0.9.7, IIRC. That forward facing air filter on the -500 was waaaay too close to not have better airflow.

Re: filters

I've been looking into adding meshes for the 40mm fans. I bought some cheap ones, but they break easily and are hard to get off. I'm hoping to find a solution where you can slip the filter in and out from behind or in front of the fan, i.e., not have to remove the fan itself.

What about a slide out *tray* for air filters? Or if the front fan panel come off, perhaps...?

**Reply by TheoBromine2022** [reply]
https://www.amazon.com/gp/product/B08L91FCM9/ref=ox_sc_act_title_1

I'm looking at these and wondering if a 4010 fan in the front would be adequate. I think the order would be box, filter, fan. and the little clip on thingie would just be tossed. I think there is enough room there, and if there isnt it wouldnt be a complete loss because the top of the box needs a 40mm filter anyway.

**Reply by jon-harper** [reply]
Those are the ones I tried. The filters are okay, but the holders snap easily when you open them. I figured I'd keep the filters and print a new backing that doesn't break so easily, but got careless and tossed the filters, too. 🤦‍♂️ Some kind of little tray or slot for those filters to clean them without removing the fan is where my mind is headed.

Is the RSP-500's fan an exhaust or intake by default, btw? I figured intake, but never asked.

**Reply by TheoBromine2022** [reply]
Exhaust

**Comment by TheoBromine2022** [comment]
Yeah I guess I really need to look at the fan orientation but I made an
assumption that it was an intake... yikes. It's a 4020 fan so push comes to
shove I'll flip it around to make it an intake. I'll see what  clearance I
have, I have a spare 4010 noctua laying around that would be good for this.
I'll post photos later.

**Reply by jon-harper** [reply]
Awesome. Thanks for checking.

**Reply by TheoBromine2022** [reply]
No problem. Now I gotta stop myself from trying to color coordinate the jst connectors for the steppers and switches. I think it would be dope to have a main bus connector on the printer and on the back of the box and have stuff plug into both so that all i have is the omnibox, a thick cable, and the printer.

**Reply by jon-harper** [reply]
We are on one wavelength here. I've got enclosed Ender 5s down to four connectors at the rear panel using Micro Fit 3. 40+ pin connectors aren't normal, so I've gotta divvy them up somehow (20 pins for the toolhead, 16+6 for motion and lights, and another 6 pin for the enclosure thermistor and fans).

I'm in the process of upgrading a second case to 0.9.9 and it's a case that I color coded the wires for. I'm having the same problem of wanting to go back and color code my others now. 😂

![image](https://user-images.githubusercontent.com/11852387/200023467-3ec81aed-ee22-435f-b28c-f5046acbfaec.png)

**Reply by TheoBromine2022** [reply]
https://www.ebay.com/itm/123694177995

I saw these while trying to find premade wires and I was like... well... maybe red for one axis going into the motherboard and one going into the switch... and then I kept digging and saw they had the 4 pins for the steppers too... so why not just color coordinate the axis switch with the axis stepper motor with the same color?

**Reply by jon-harper** [reply]
I like the idea of color coded endpoint connectors. I would even jump on that if Micro Fit 3 had color options.

Color coded wire tags are an option if you don't want to replace connectors. I've seen those and just haven't switched over from P-Touch labels with letter codes for my own wiring.

**Comment by jon-harper** [comment]
So I considered this possibility, actually. Another redditor that initially
requested the RSP-500 support also wanted a rear panel.

I designed the panel with an 80mm fan that is an *intake* and reverses the
usual direction of airflow. It's mounted low to split airflow between the
PSU and rest of the case.

It's in Rear Panels/Custom/Ender 5 Plus, I believe.

That would also give you a single fan to provide a filter for...

**Reply by TheoBromine2022** [reply]
![20221104_164757](https://user-images.githubusercontent.com/111667850/200072896-1f681415-3fec-430d-99b7-ca9fb26f84a4.jpg)
![20221104_165245](https://user-images.githubusercontent.com/111667850/200072921-7fe82b77-7b4d-4a37-81d9-7c87a1410c64.jpg)


Quick and dirty e5+ base risers. M3-.5mm tap. Took 5 minutes.

**Reply by jon-harper** [reply]
Wait, you're missing the base extensions! That's why it's a tight fit! There are two `STL` files under core that add about 7mm or so to the base.

**Comment by TheoBromine2022** [comment]
Oh I know. That's why this is the cheap and dirty method. It doesn't need
but 2-3mm

**Comment by jon-harper** [comment]
Lol. Beautiful.

**Comment by TheoBromine2022** [comment]
![20221104_212051](https://user-images.githubusercontent.com/111667850/200094823-2c829d4e-2183-47f4-931d-504282e7de57.jpg)
This screw is basically impossible without some sort of cutout in the base of the fan mount.

![20221104_212023](https://user-images.githubusercontent.com/111667850/200094852-37a22083-325c-4ea0-8896-346947256a44.jpg)
The power supply actually fits fairly flush with the base of the stock bottom print. Using m3 machine screws is adequate.

![20221104_212005](https://user-images.githubusercontent.com/111667850/200094888-cfebfc06-52f9-43cb-a4e1-37a6348a457f.jpg)
Verdict: 10 pounds of shit in a 5 pound bag. Looking at the monolithic prints there is about 3 more cm I can play with to increase the length and keep it one solid print. I think I will take advantage of that and extend the length to accommodate fan space and wiring clearance.

**Reply by jon-harper** [reply]
Whoosh.

Gonna sleep on this one and look at it in the AM. I'll come up with something...

**Comment by jon-harper** [comment]
I had an idea that I think may work, basically by taking yours and running with it. [Possible fix instructions under the issue](https://github.com/jon-harper/OmniBox/issues/60).

**Reply by TheoBromine2022** [reply]
I think sliding it all the way to the front of the case and having the fan port line up with the psu exhaust fan and keeping the rear panel open to using the 80mm would be the easiest fix . goes against everything conventional ive done with computers having the exhaust vent straight at the user but it would work.

edit: this opens up options to using an 80mm filter on the rear

**Reply by jon-harper** [reply]
I see where you're coming from. That saves a LOT of trouble managing two (edit: more) base designs. This is the 0.9.9 base and RSP-500 PSU mount without any modifications. The terminal area in the back now has as much room as the Mean Well LRS-350 has. Would you rather it be all the way up to the front, or is this enough?

![image](https://user-images.githubusercontent.com/11852387/200127854-211b3339-c013-4d20-9740-4a8ae5874b04.png)

**Reply by TheoBromine2022** [reply]
ooo. yeah moving the switch vertical would make it bridgeable. the power supply would definitely require that if it was to be moved all the way forward. are you setting it up so that the power supply has its own cradle? im trying to interpret the red object. I think in the case of the rp500 it would be good if the back of it was as flush to the front panel as possible. there are also two screws on the back of the rp500 going into the fan that stick out from the psu enclosure by 3mm. 7mm diameter

![20221105_115059](https://user-images.githubusercontent.com/111667850/200128509-977f8a11-28db-47e1-bcfc-4fd78479a747.jpg)

**Comment by jon-harper** [comment]
> are you setting it up so that the power supply has its own cradle? 

Yep yep. That was the plan for 0.9.9 all along. Let me show you ~~want~~ what I already have set up and let me know what you think:

----

Power supplies are now a fourth type of tray and can be replaced without removing stuff from the top of the case. 😉

Here's a different angle. There are now three vents up front, too.

![image](https://user-images.githubusercontent.com/11852387/200129639-3f84f695-df16-41bf-978b-2ea9b82a2f2b.png)

This is the 0.9.9 unified base (that's added to 0.9.9, too) and the base extension shims. The shims sit between the base and the the bottom panel and give extra room for the RSP-500. It has about 3mm above and below for airflow.

![image](https://user-images.githubusercontent.com/11852387/200128920-e118b6fc-3845-46bb-9317-9df5413f236b.png)

And another picture so you can see it with the new bottom panel. The screws for the bottom panel pass through the base extension into the screw holes you've used on the base before. (There is also a completely closed version of this panel if you'd like.)

----

Summary, in the [0.9.9 branch](https://github.com/jon-harper/OmniBox/tree/0.9.9):

1. `Trays/PSU/Mean Well RSP-500` has two STLs for the PSU tray.
2. `Core/Base/Unified` has the unified base.
3. `Core/Base/Extension Shim` has the extension shims.
4. `Panels/Bottom Panel` has two bottom panels to pick from.

Finally, this will work with your existing main body; you'll just use this base instead.

**Reply by TheoBromine2022** [reply]
my concern is that with all of those hex holes you are allowing dust to ingress into the box, especially right behind an 80mm fan, and they are going to get blown directly into the power supply. so yeah a closed panel would probably be preferable. I feel like the psu being flush would provide better airflow for the exhaust fan, otherwise youd be potentially blowing hot air straight through the vent holes below the display. it would also make those vents rather pointless if the psu is directly up against the front, so you could just treat it as infill. I think lining up the middle fan port with the rsp500 would be the best bet, but now you are entering the territory of making an omnibox specifically for e5+... which... I am not against doing the digital modifications for in the future on your revisions.

Im keeping this omnibox, as its functional, but I 100% intend to modify what you publish and send back to you. I intend to start a print farm so Id be making quite a few of them.

**Reply by jon-harper** [reply]
> so yeah a closed panel would probably be preferable.

Agreed, the closed panel makes more sense in your case. I just didn't have the model loaded at the moment, so I showed with the hex panel. They're both in the folder.

> Im keeping this omnibox, as its functional, but I 100% intend to modify what you publish and send back to you.

You're definitely welcome to modify! So far every idea you've had has been great. As you move forward printing cases, I'd love to see what you come up with and hear your feedback.

> I intend to start a print farm so Id be making quite a few of them.

Since you're printing more OmniBoxes, can I recommend that you base your changes on the 0.9.9 Core parts? I don't know how much you've looked at it, but they're just...better cases. The fan mount/floor screw issue that you found is fixed (and the very front two screws behind the display are also easier to reach). Plus a lot of other fixes and improvements. It'll also be much easier to integrate your work.

Again, I really appreciate your feedback and time.

**Reply by TheoBromine2022** [reply]
Oh yeah the modifications i sent you were probably only 2-3 hours worth of dicking around with the files I already had, I await the next release, or one you are content with for quite a while. i'll get serious about modifying those

**Reply by jon-harper** [reply]
Sweet, thanks. That release is coming soon here. 🙂

**Comment by TheoBromine2022** [comment]
Also... on the subject of power supply cable management.

Since the power supply is what grounds the printer, are you running a ground wire to the power supply from the printer? Also it made me realize that the steppers are all isolated with rubber wheels so wouldn't  each one would need a ground wire?

**Reply by jon-harper** [reply]
The bed is the only thing that really needs grounding, and the frame only needs grounding because it's touching the bed. The thing about earthing is that you're only trying to prevent *yourself* from getting shocked. If a short doesn't trip a fuse, that wire keeps you from becoming a method electrical discharge. So the wire needs the shortest path it can get to the bed (shorter than human touching frame).

Ideally, Creality would have included an earthing wire with the bed, particularly on the E5+'s monster bed. They didn't, so the next best bet is a 16AWG wire attached close to the bed to metal. The Ender 5 Pro has a bunch of screws holding the bed to the cantilever, so that's my go-to and I run with the bed wiring. You can use a ring connector and use the screw head as the "terminal".
